examples/fineFoods/admin/mui/src/components/layout/header/index.tsx

```diff
@@ -26,21 +148,127 @@ export const Header: React.FC<HeaderProps> = ({
+                            disableClearable
+                            options={options}
+                            filterOptions={(x) => x}
+                            fullWidth
+                            size="small"
+                            sx={{ maxWidth: 550 }}
+                            onInputChange={(event, value) => {
+                                if (event?.type === "change") {
+                                    setValue(value);
+                                }
+                            }}
+                            groupBy={(option) => option.category}
+                            renderOption={(props, option: IOptions) => {
+                                return (
+                                    <Link href={option.link} underline="none">
+                                        <Box
+                                            {...props}
+                                            component="li"
+                                            sx={{
+                                                display: "flex",
+                                                padding: "10px",
+                                                alignItems: "center",
+                                                gap: "10px",
+                                            }}
+                                        >
+                                            <Avatar
+                                                sx={{ width: 64, height: 64 }}
+                                                src={option.url}
+                                            />
+                                            <Typography
+                                                style={{ fontSize: "16px" }}
+                                            >
+                                                {option.label}
+                                            </Typography>
+                                        </Box>
+                                    </Link>
+                                );
+                            }}
+                            renderInput={(params) => {
+                                return (
+                                    <TextField
+                                        {...params}
+                                        label={t("search.placeholder")}
+                                        InputProps={{
+                                            ...params.InputProps,
+                                            type: "search",
+                                        }}
+                                    />
+                                );
+                            }}
+                        />
+                    </Stack>
+                    <Stack direction="row" gap="20px" alignItems="center">
+                        <FormControl sx={{ m: 1, minWidth: 120 }}>
+                            <Select
+                                defaultValue="en"
```

Comment:
```suggestion
                                defaultValue="en"
```
Shouldn't we be doing this dynamically?

Reply:
yep, we can get the current locale for defaultValue. I am going to fix it. 🤸‍♀️